### PR TITLE
[Cloud Security] add the cloud formation credentials url

### PR DESCRIPTION
**Pull request by Omolola-Akinleye**
## Proposed commit message
Add cloud formation credentials template url which help users create stack to automate direct access keys to copy and paste to input fields

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates [#175977](https://github.com/elastic/kibana/issues/175977)

## Screenshots
image

### CODE REVIEW

**Review by seanrathier — APPROVED** [review]
Approved with a couple of comments

**Comment by orouz** [comment]
@Omolola-Akinleye 

i talked to @maxcold about this a while ago and i think if it's possible to just store these values in kibana that'd be preferable

File: packages/cloud_security_posture/manifest.yml [hunk 1]
```diff
@@ -116,6 +116,15 @@ policy_templates:
...
             default: https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://elastic-cspm-cft.s3.eu-central-1.amazonaws.com/cloudformation-cspm-ACCOUNT_TYPE-8.14.0.yml&stackName=Elastic-Cloud-Security-Posture-Management&param_EnrollmentToken=FLEET_ENROLLMENT_TOKEN&param_FleetUrl=FLEET_URL&param_ElasticAgentVersion=KIBANA_VERSION&param_ElasticArtifactServer=https://artifacts.elastic.co/downloads/beats/elastic-agent
+          - name: cloud_formation_credentials_template
+            type: text
+            title: CloudFormation Credentials Template
+            multi: false
+            required: true
+            show_user: false
+            description: Template URL to create cloud credentials to Create Stack
+            # ACCOUNT_TYPE value should be either "single-account" or "organization-account"
+            default: https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://elastic-cspm-cft.s3.eu-central-1.amazonaws.com/cloudformation-cspm-direct-access-key-ACCOUNT-TYPE-8.14.0.yml
```
Comment (orestisfl) [inline packages/cloud_security_posture/manifest.yml:127]:
```suggestion
            default: https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://elastic-cspm-cft.s3.eu-central-1.amazonaws.com/cloudformation-cspm-direct-access-key-ACCOUNT_TYPE-8.14.0.yml
```

Notice the underscore!

File: packages/cloud_security_posture/manifest.yml [hunk 2]
```diff
@@ -116,6 +116,15 @@ policy_templates:
             description: Template URL to Cloud Formation Quick Create Stack
             # ACCOUNT_TYPE value should be either "single-account" or "organization-account"
             default: https://console.aws.amazon.com/cloudformation/home#/stacks/quickcreate?templateURL=https://elastic-cspm-cft.s3.eu-central-1.amazonaws.com/cloudformation-cspm-ACCOUNT_TYPE-8.14.0.yml&stackName=Elastic-Cloud-Security-Posture-Management&param_EnrollmentToken=FLEET_ENROLLMENT_TOKEN&param_FleetUrl=FLEET_URL&param_ElasticAgentVersion=KIBANA_VERSION&param_ElasticArtifactServer=https://artifacts.elastic.co/downloads/beats/elastic-agent
+          - name: cloud_formation_credentials_template
+            type: text
+            title: CloudFormation Credentials Template
+            multi: false
+            required: true
+            show_user: false
+            description: Template URL to create cloud credentials to Create Stack
```
[the remark this answers is not in the document]
Comment (Omolola-Akinleye) [reply packages/cloud_security_posture/manifest.yml:125]: I will change it  to  `Template URL to Cloud Formation Cloud Credentials Stack`  similar description structure to `Template URL to Cloud Formation Quick Create Stack`

File: packages/cloud_security_posture/manifest.yml (hunk 1 above)
Comment (Omolola-Akinleye) [reply packages/cloud_security_posture/manifest.yml]: Nice catch! Thanks @orestisfl

**Comment by Omolola-Akinleye** [comment]
> @Omolola-Akinleye
> 
> i talked to @maxcold about this a while ago and i think if it's possible to just store these values in kibana that'd be preferable



> @Omolola-Akinleye
> 
> i talked to @maxcold about this a while ago and i think if it's possible to just store these values in kibana that'd be preferable
>>  Hey @orouz I missing some context. Would you remind me why we want to store these values in Kibana?  I discussed with @kfirpeled  and we decided to use Integration so when the package version is being upgraded along with the template url version will be upgraded.

**Comment by oren-zohar** [comment]
@Omolola-Akinleye I think @orouz raise an interesting point and we should discuss it offline, but it's out of this PR scope, so feel free to merge if you think it's ready

**Review by orestisfl — APPROVED** [review]

**Comment by Omolola-Akinleye** [comment]
> @Omolola-Akinleye I think @orouz raise an interesting point and we should discuss it offline, but it's out of this PR scope, so feel free to merge if you think it's ready

Thanks @oren-zohar  @orouz I'll  merge for now, and create a thread to discuss offline

**Pull request merged**